src/views/shortcodes/my-attendance-list.php

```diff
@@ -22,6 +22,9 @@
 					<span class="datetime">(<?php echo $start_date; ?>)</span>
 				<?php endif; ?>
 			</a>
+			<a href="<?php echo esc_url( get_permalink( $id ). 'tickets' ); ?>" target="_blank">
+                		- <span><?php esc_html_e( 'View Tickets', 'event-tickets' ) ?></span>
+            		</a>
```

Comment:
:no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: ce7e6169731d6120c9fc57e6f21197ad</sub>